sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/PredicateSuite.scala

```diff
@@ -455,4 +456,10 @@ class PredicateSuite extends SparkFunSuite with ExpressionEvalHelper {
     interpreted.initialize(0)
     assert(interpreted.eval(new UnsafeRow()))
   }
+
+  test("[SPARK-24872] Replace the symbol '||' of Or operator with 'or'") {
```

Comment:
tiny nit: `[SPARK-24872] ` -> `SPARK-24872: `

Reply:
Thank you! I have made changes to the code. @HyukjinKwon